### ISSUE TITLE
Loot table issue BWL/MC

### ISSUE THREAD

**Issue by Mofoleone** [FeenixServerProject/Phoenix_1.12.1_Issue_tracker#5150]
Since no one reported here, let me post whats the problem:

Molten Core:

Garr = Bindings of the Blessed Windseeker
Baron Geddon = Bindings of the Blessed Windseeker
Ragnaros = Bonereaver's Edge / Onslaught Girdle / Spinal Reaper

Blackwing Lair:

Vaelastrasz = Red Dragonscale Protector
Broodlord = Maladath, Runed Blade of the Black Flight / Heartstriker
Ebonroc = Drake Fang Talisman / Dragonbreath Hand Cannon
Flamegor = Styleen's Impending Scarab
Chromagus = Chromatically Tempered Sword / Empowered leggins
Nefarian = Ashkandi, Greatsword of the Brotherhood / Crul'shorukh, Edge of Chaos

Zul'Gurub:

Jin'do the Hexxer = Jin'do's Hexxer / Jin'do's Evil Eye
High Priestess Arlokk = Will of Arlokk

**Comment by Kaiitothewai** [comment]
Didn't plenty of people state they had many of 1x binding but 0x of the other half? or is this based on your personal experience

**Comment by bunnehhh** [comment]
Karabaja is this your wishlist? Maybe at least one binding should drop every MC?

**Comment by Mofoleone** [comment]
@Kaiitothewai personal guild experience no bindings for 7-8 months, people are being superstitious about who will make id/lead to get them as drop (like that matter)
Drop % are once again broken, but can't be that hard to check them out.

@bunnehhh if you don't have anything to add/contribute, please stay away from bugtracker, this isn't place for tea party/discussions.

**Comment by Nzzoth** [comment]
I would like to remind you that this is not a forum, discussions that are held here are strictly about the issue at hand.

**Comment by Kaiitothewai** [comment]
I don't think so

**Comment by Mofoleone** [comment]
This report was made purely based on:
- personal experience (that should matter the most in this case)
- comments and suggestions from other people on Emerald Dream/Warsong (trusted people)
- realmplayers.com site that provides informations about players using specific items (third party site, but still trustable)
And I made it on request from this thread:
http://www.wow-one.com/forum/topic/93999-corrupted-ashbringer-is-removed

**Comment by ehodzic** [comment]
Some guilds get none of one item, some get none of other item (for a long period); but when you put it all together, everything is dropping.

**Comment by wlfbck** [comment]
@ehodzic That is probably true. Ofcourse there is a good chance it's simply luck, but it would be interesting to see how drops are determined and if their is a connection to players participating in the raid.

**Comment by Mofoleone** [comment]
Updatin issue with aq40/naxxramas, info from thread, from Snelf and some other warsong players.
Naxxramas: Instructor Razuvious - Iblis
4hm - Corrupted Ashbringer
Maexxna - Wraith Blade. Haven't seen it in... months.

AQ40:	C'thun - Axe, Plate Gloves. Nowadays it mostly drops cloaks and belts.

BWL:	Drakefang Talisman, Styleen's trinkets have ridiculously low drops.

AQ20:	Ossirian - the crossbow.

Items with higher drop rate:
Naxxramas: 4hm - leather pants, mp5 trinket. 



Would be nice to get some love here from mods on drop rates, since it's frustrating to clear same content all over and over again with no proper "rewards".

**Comment by Cabro** [comment]
Yes right now, clearing AQ40 is pointless since Cthun only  drops healing cloak and belt and ring. Nothing else. Similar to how Nefarian was only dropping leather boots, mail belt and caster cloak. Useless drops.

**Comment by Kaiitothewai** [comment]
Sounds like there's some changes from the ones Database made back in start of 2014.

**Comment by Mofoleone** [comment]
Information gained for the first post I made in this report are made based on several guilds on Emerald Dream and my own personal. More info can be found if you look at drops @realmplayers site.
Drop rates of weapons/shields/trinkets and certain items are lowered and that's a fact.

**Comment by Redtempest** [comment]
To chime in on the whole Binding-Drop-Rate-Stuff - surely personal experience :
Since we are raiding with Endurance (my first raid with the guild was on 12.12.12 - they started 2-3 weeks before i joined) we had 7 Garr Bindings and 0 Geddon Bindings. The one Geddon Binding we have used for the TF of our tank Arawn, was obtained from the time when he was still in Equilibrium.

In all that time we only skipped 3, maybe 4 MC IDs (Christmas and stuff). 
And even our last Garr Binding dropped more than one year ago.

Take that bit of information however you like, but IMHO it is worth looking into this.

**Comment by Sorrugis** [comment]
I agree that drop rates appear to be bugged for our guild on most of the listed bosses that I've experienced above.

Our highest priority rogue had to wait a full year for his Drakefang Talisman. Probably been a solid 9-10 months since we've seen a Styleens. Also, we haven't seen a shield drop off Chromaggus in what I'd guess to be over a year and a half. I also agree that the mail belt and caster cloak have much higher drop rates than they should off Nefarian.
